loadtestconfig.default.json

```diff
@@ -75,7 +75,11 @@
         "UserProfileUpdateNicknameChance": 0.01,
         "UserProfileUpdatePositionChance": 0.107,
         "UserProfileUpdateEmailChance": 0.003,
-        "UserProfileUpdateImageChance": 0.865
+        "UserProfileUpdateImageChance": 0.865,
+        "PublicChannelCreationChance": 0.12,
```

Comment:
Where are these numbers from?

Reply:
The `UserProfile` part comes from some analytics data I was given specific to this case.
The values in the `ChannelCreation` section were calculated from the default config itself using `NumChannelsPerTeam` and other related values.  In this case I didn't have access to any relevant statistics.